tutors/views.py

```diff
@@ -1,177 +1,125 @@
-from .models import CategoryQuestion, Question, Answer
-from .forms import CategoryCreateForm, QuestionCreateForm, AnswerCreateForm
-from quiz.models import  Quizzes
-
-ITEMS_PER_PAGE = 5
-
-
-class CategoryListView(LoginRequiredMixin, ListView):
-    model = CategoryQuestion
-    template_name = 'tutors/categories_list.html'
-    context_object_name = 'categories'
-    paginate_by = ITEMS_PER_PAGE
-
-    def get_queryset(self):
-        """
-        Returns the queryset of categories that you want to display.
-        """
-        categories = CategoryQuestion.objects.all().annotate(
-            num_question=Count('question')).order_by('pk')
-        q = self.request.GET.get("category_search")
-        if q:
-            return categories.filter(name__icontains=q)
-        return categories
-
-    def get_context_data(self, **kwargs):
-        """
-        Returns context data for displaying the list of categories.
-        """
-        context = super().get_context_data(**kwargs)
-        context['q'] = self.request.GET.get("category_search")
-        return context
-
-
-class CreateCategoryView(LoginRequiredMixin, SuccessMessageMixin, CreateView):
-    model = CategoryQuestion
-    form_class = CategoryCreateForm
-    template_name = 'tutors/category_create.html'
-    success_url = reverse_lazy('tutors:categories_list')
-    success_message = 'Category: "%(name)s" was created successfully'
-
-    def form_valid(self, form):
-        obj = form.save(commit=False)
-        obj.save()
-        return super().form_valid(form)
-
-
-class UpdateCategoryView(LoginRequiredMixin, SuccessMessageMixin, UpdateView):
-    model = CategoryQuestion
-    form_class = CategoryCreateForm
-    template_name = 'tutors/category_create.html'
-    success_url = reverse_lazy('tutors:categories_list')
-    success_message = 'Category: "%(name)s" was updated successfully'
-
-    def get_success_url(self):
-        return reverse_lazy('tutors:categories_list')
-
-    def get_form_kwargs(self):  # defined in ModelFormMixin class
-        """
-        Returns the keyword arguments for instantiating the form.
-        """
-        kwargs = super(UpdateCategoryView, self).get_form_kwargs()
-        if hasattr(self, 'object'):
-            kwargs.update({'instance': self.object})
-        return kwargs
-
-
-class DeleteCategoryView(LoginRequiredMixin, SuccessMessageMixin, DeleteView):
-    model = CategoryQuestion
-    template_name = 'tutors/category_delete.html'
-    success_url = reverse_lazy('tutors:categories_list')
-    success_message = 'Category: "%(name)s" was deleted successfully'
-
-    def delete(self, request, *args, **kwargs):
-        """
-        Returns context data about success deleted category in message.
-        """
-        obj = self.get_object()
-        messages.success(self.request, self.success_message % obj.__dict__)
-        return super().delete(request, *args, **kwargs)
+from CultureAnalyzer.settings.default import ITEMS_ON_PAGE
```

Comment:
According to [pep-8](https://www.python.org/dev/peps/pep-0008/#imports) a blank line before `from CultureAnalyzer...` is required